api/turing/webhook/constant.go

```diff
@@ -0,0 +1,41 @@
+package webhook
+
+import (
+	"errors"
+
+	"github.com/caraml-dev/mlp/api/pkg/webhooks"
+)
+
+var (
+	OnRouterCreated    = webhooks.EventType("OnRouterCreated")
```

Comment:
the event type format is different with the one in merlin, can we agree on the same format?

Reply:
Fixed on https://github.com/caraml-dev/turing/pull/395/commits/5ac941e0028de90e2cea6279ceb9fc89c7626b52, thank you for your detail 🙏